tempesta_fw/t/functional/tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+__author__ = 'Tempesta Technologies Inc.'
+__copyright__ = 'Copyright (C) 2016 Tempesta Technologies Inc. (info@natsys-lab.com).'
+__license__ = 'GPL2'
+
+__all__ = [ "bomber", "fragmented_requests", "test_frang", "test_cache",
+	    "test_parser" ]
```

Comment:
Ok, now I see where I can find the list of tests. I'd be cool to make execution scripts (`tests.py` and/or `run_tests.sh` to print the tests by some argument, i.e. `--list`).